model.py

```diff
@@ -551,3 +555,48 @@ def calc_mutations_per_tree(self):
         mutations_per_tree = np.zeros(self.ts.num_trees, dtype=np.int64)
         mutations_per_tree[unique_values] = counts
         return mutations_per_tree
+
+    def compute_ancestor_spans_heatmap_data(self, win_x_size=1_000_000, win_y_size=500):
```

Comment:
Are these sizes of genome coordinates? If so, it's going to be tricky to make default that work across species (compare humans to sars 2, eg.).

What you using number of x-bins and number of y-bins instead? That way you can be sure of a given level of resolution and that you won't use too much memory.

Can use np.linspace to generate the bin coordinates.